EmployeeService.Tests/EmployeeServicesTest.cs

```diff
@@ -1,36 +1,47 @@
 using System;
 using Xunit;
-using EmployeeService.Tests;
-using Moq;
 using EmployeeServices.Model;
-using EmployeeServices.Services;
 using EmployeeServices.Controllers;
 using System.Collections.Generic;
 using Microsoft.AspNetCore.Mvc;
+using Moq;
+using EmployeeServices.Services;
 
 namespace EmployeeService.Tests
 {
     public class EmployeeServicesTest
     {
         private readonly EmployeeServicesController _employeeController;
-        
+        private Mock<IEmployeeService> _mockEmployeesList;
+
+        public EmployeeServicesTest()
+        {
+            _mockEmployeesList = new Mock<IEmployeeService>();
+            _employeeController = new EmployeeServicesController(_mockEmployeesList.Object);
+        }
+
         [Fact]
-        public void Get_WhenCalled_ReturnsOkResult()
+        public void GetTest_ReturnsListofEmployees()
         {
-            // Act
-            var okResult = _employeeController.Get();
-            // Assert
-            Assert.IsType<OkObjectResult>(okResult.Result);
+           
+            var result = _employeeController.Get();
+            Assert.IsAssignableFrom<ActionResult<List<Employee>>>(result);
+
+
         }
+
         [Fact]
-        public void Get_WhenCalled_ReturnsAllItems()
+        public void Get_ActionExecutes_ReturnsExactNumberOfEmployees()
         {
-            // Act
-            var okResult = _employeeController.Get().Result as OkObjectResult;
-            // Assert
-            var employees = Assert.IsType<List<Employee>>(okResult.Value);
-            Assert.Equal(3, employees.Count);
+            _mockEmployeesList.Setup(repo => repo.GetAllEmployees())
+               .Returns(new List<Employee>() { new Employee(), new Employee() });
+            var result = _employeeController.Get();
+            var viewResult = Assert.IsAssignableFrom<ActionResult<List<Employee>>>(result);
+            var employees = Assert.IsType<List<Employee>>(viewResult.Value);
+            Assert.Equal(2, employees.Count);
```

Comment:
count is not a good enough test, we will need to use equal on actual data
Assert.Equal(expected, actual)
or via fulent assertions actual.ShoulBeEquivalentTo(expected)